docs/_changelog/header/v0.11.6.md

```diff
@@ -187,18 +187,18 @@ class Program
 }
 ```
 
-You can achieve the same thing using `[TargetFrameworkJobAttribute]`:
+You can achieve the same thing using `[SimpleJobAttribute]`:
 
 ```cs
 using BenchmarkDotNet.Attributes;
 using BenchmarkDotNet.Jobs;
 
 namespace BenchmarkDotNet.Samples
 {
-    [TargetFrameworkJob(TargetFrameworkMoniker.Net48)]
-    [TargetFrameworkJob(TargetFrameworkMoniker.Mono)]
-    [TargetFrameworkJob(TargetFrameworkMoniker.NetCoreApp21)]
-    [TargetFrameworkJob(TargetFrameworkMoniker.NetCoreApp30)]
+    [SimpleJob(runtimeMoniker: RuntimeMoniker.Net48)]
```

Comment:
Don't like the `runtimeMoniker` prefix. Could we introduce a few additional `SimpleJobAttribute` ctor overloads with the `RuntimeMoniker` as the first ctor argument?